src/typedtree.ml

```diff
@@ -574,6 +579,7 @@ module Expr = struct
     | Prm x            -> P.prim (Prim.untype x)
     | Uno (Fst, x)     -> P.fst (untype x)
     | Uno (Snd, x)     -> P.snd (untype x)
+    | Uno (Get i, x)   -> P.get i (untype x)
```

Comment:
Making `Get` a unary operator that holds the index means that the index must be statically known. So, it's impossible to write something like this:
```
\ (i : Int) -> \ (xs : List Bool) -> Get i xs
```
(Unless I'm missing something)
What's the reasoning behind that?

If `Get` was a binary operator taking the index and the list as arguments, then dynamic indices would be possible.

Reply:
That was a quick hack to test the demo app, I'll push something better. Thanks for noticing :-)